packages/UI/snapshots.yml

```diff
@@ -0,0 +1,2 @@
+- name: Local Storybook atom snapshots
+  url: http://localhost:6006
```

Comment:
what does this file do? where is it taken from?

Reply:
This is for taking snapshots with Percy through a different way. It was taking from reading Percy's documentation.

Reply:
is it still needed? when you say different way does that mean non CI way? if so can we get remove it

Reply:
You can use it locally or with CI/CD. Either way, I removed it.